specification/contosowidgetmanager/data-plane/readme.md

```diff
@@ -0,0 +1,26 @@
+# Contoso.WidgetManager
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Contoso.WidgetManager.
+
+## Configuration
+
+### Basic Information
+
+This is a TypeSpec project so we only want to readme to default the default tag and point to the outputted swagger file.
+This is used for some tools such as doc generation and swagger apiview generation it isn't used for SDK code gen as we
+use the native TypeSpec code generation configured in the tspconfig.yaml file.
+
+```yaml
+tag: package-2022-11-01-preview
+```
+
+### Tag: package--2022-11-01-preview
+
+These settings apply only when `--tag=package--2022-11-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2022-11-01-preview'
+input-file:
+  - Azure.Contoso.WidgetManager\preview\2022-11-01-preview\widgets.json
```

Comment:
I think backward slash here confuses swagger API parser. I will update it and check.

Reply:
Yes. I will submit a PR to ignore tspconfig.yaml from swagger review files. I think I have filed an issue for this change.

Reply:
@praveenkuttappan it does look like the path seperators were the issue for the swagger generation.